lib/Runtime/Executor/ThreadPoolExecutor.cpp

```diff
@@ -372,42 +371,33 @@ void ThreadPoolExecutor::executeDAGNode(
 
   // If tracing is enabled, set the thread name for TraceEvents for this node to
   // be the name of the Device.
-  if (executionState->getRawResultContextPtr()->getTraceContext()) {
-    executionState->getRawResultContextPtr()->getTraceContext()->setThreadName(
-        currentDevice, deviceManager->getDeviceConfig()->getName());
-  }
+  // if (executionState->getRawResultContextPtr()->getTraceContext()) {
```

Comment:
Ooops.

Reply:
I do have the TODO for this below, but I could move it up so it's clearer. Intentionally commented out because I want to fix the thread names.